gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2/api-creation-v4-step-2.component.html

```diff
@@ -67,3 +72,5 @@ <h3 class="mat-h3 api-creation-v4-step-2__banner__title">
     </div>
   </form>
 </mat-card>
+
+<ng-template #loadingTpl><mat-progress-bar mode="indeterminate"></mat-progress-bar></ng-template>
```

Comment:
📝 mat-progress-bar is not the final spinner/progress. It's the temporary one. We wait the one from UI/UX team